packages/cells/src/widget.ts

```diff
@@ -1061,6 +1088,45 @@ export class CodeCell extends Cell<ICodeCellModel> {
     this.node.setAttribute('aria-label', ariaLabel);
   }
 
+  getOutputPlaceholderText(): string | undefined {
```

Comment:
This should not be public

```suggestion
  protected getOutputPlaceholderText(): string | undefined {
```